CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [0.1.1]
+
+-  New commands
+   -  `Build an image on Azure Container Registry`
+   -  `Prerequisites`
```

Comment:
Might want to mention that the deploy experience was also added. It's not a command but can be triggered after creating deployment files.